src/screens/WBTCCafe/WBTCCafe.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { LayoutAnimation, RefreshControl } from 'react-native';
+import styled, { withTheme } from 'styled-components/native';
+import type { NavigationScreenProp } from 'react-navigation';
+import t from 'translations/translate';
+
+// components
+import ContainerWithHeader from 'components/Layout/ContainerWithHeader';
+import CircleButton from 'components/CircleButton';
+import ActivityFeed from 'components/ActivityFeed';
+import { ScrollWrapper } from 'components/Layout';
+
+// actions
+import { hasSeenWbtcCafeIntroAction } from 'actions/appSettingsActions';
+import { fetchSmartWalletTransactionsAction } from 'actions/historyActions';
+import { setWbtcPendingTxsAction } from 'actions/exchangeActions';
+
+// utils, services
+import { getValidPendingTransactions, getWbtcCafeTransactions, mapPendingToTransactions } from 'services/wbtcCafe';
+import { getTransactionsFromHistory } from 'utils/history';
+import { getSmartWalletAddress } from 'utils/accounts';
+
+// types
+import type { Theme } from 'models/Theme';
+import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
+import type { Transaction } from 'models/Transaction';
+import type { PendingWBTCTransaction } from 'models/WBTC';
+import type { Accounts } from 'models/Account';
+
+// selectors
+import { combinedHistorySelector } from 'selectors/history';
+
+// constants
+import { WBTC, BTC } from 'constants/assetsConstants';
+import { EXCHANGE } from 'constants/exchangeConstants';
+
+import WBTCCafeIntro from './WBTCCafeIntro';
+
+interface Props {
+  navigation: NavigationScreenProp;
+  hasSeenWbtcCafeIntro: boolean;
+  updateHasSeenIntro: () => void;
+  theme: Theme;
+  history: Transaction[];
+  pendingWbtcTransactions: PendingWBTCTransaction[];
+  accounts: Accounts;
+  fetchSmartWalletTransactions: () => void;
+  setWbtcPendingTxs: (txs: PendingWBTCTransaction[]) => void;
+}
+
+interface State {
+  showIntro: boolean;
+}
```

Comment:
What's up with these `interfaces` Jan? 😂 

```suggestion
type Props = {
  navigation: NavigationScreenProp,
  hasSeenWbtcCafeIntro: boolean,
  updateHasSeenIntro: () => void,
  theme: Theme,
  history: Transaction[],
  pendingWbtcTransactions: PendingWBTCTransaction[],
  accounts: Accounts,
  fetchSmartWalletTransactions: () => void,
  setWbtcPendingTxs: (txs: PendingWBTCTransaction[]) => void,
};

type State = {
  showIntro: boolean,
}
```